# Configuration file for the microphone
microphone:
  default_param:
    audio_format_width: 2
    chunk_size: 1024
    total_channels: 1
    audio_rate: 16000
    device_name: "default"
    test_outdir: "$(find harmoni_microphone)/temp_data/test_example.wav"
